ansible/files/paper-config/plugins/ExilePearl/config.yml

```diff
@@ -260,6 +260,7 @@ general:
   max_pearled_alts: 1
   max_pearled_message: 'You have too many imprisoned alts'
   can_elytra_with_pearls: true
+  can_pearled_players_pearl_others: false
```

Comment:
Might be worth setting this to true, as then the feature can be merged without any actual change to the game, which can be a different discussion.

Reply:
very true, default value is now true